### ISSUE TITLE
python manage.py fake_data error

### ISSUE THREAD

**Issue by muskox** [codeforanchorage/collective-development#12]
When follow the README, and I run
`python manage.py fake_data`

I get 

```
(venv)cows:collective-development spustay$ python manage.py fake_data
Traceback (most recent call last):
  File "manage.py", line 3, in <module>
    from app import create_app
  File "/Users/spustay/gits/collective-development/app/__init__.py", line 13, in <module>
    from app import config
  File "/Users/spustay/gits/collective-development/app/config.py", line 4, in <module>
    class base_config(object):
  File "/Users/spustay/gits/collective-development/app/config.py", line 14, in base_config
    'port': int(os.environ.get('DB_PORT_27017_TCP_PORT')),
TypeError: int() argument must be a string or a number, not 'NoneType'
(venv)cows:collective-development spustay$
```

**Comment by thermokarst** [comment]
Related to #2. Development environment assumes you are running in a docker container.

**Comment by thermokarst** [comment]
@muskox: does the fix in PR #13 work for you? I tested in Docker, and in a plain virtualenv and things looked okay from my end. Thanks!

**Comment by muskox** [comment]
I am having local docker issues. On `docker-compose start` I get `Couldn't connect to Docker daemon at http+docker://localhostunixsocket`. Github and stackoverflow have ideas, the easiest being `sudo`. Not working at the moment, but I like PR #13 I would go for it,
